code/modules/projectiles/ammo_datums.dm

```diff
@@ -1075,6 +1126,7 @@ GLOBAL_LIST_INIT(no_sticky_resin, typecacheof(list(/obj/item/clothing/mask/faceh
 	damage = 85 // this is gotta hurt...
 	max_range = 40
 	penetration = 100
+	sundering = 100
```

Comment:
this will instantly remove **all** armor from whatever xeno is hit.

Reply:
I know that! I've tested it! It's a DEATHSQUAD gun! It's already penetrating fully. It's more of a showing to the player that they should run. Really really FAST.